test/e2e/rules_test.go

```diff
@@ -117,7 +117,9 @@ func TestRulesAPI(t *testing.T) {
 
 		body, err := ioutil.ReadAll(res.Body)
 		bodyStr := string(body)
+
 		assertResponse(t, bodyStr, "alert: HighRequestLatency")
+		assertResponse(t, bodyStr, "job:request_latency_seconds:mean5m{job=\"myjob\",tenant_id=\""+defaultTenantID+"\"}")
```

Comment:
Maybe adding a more complex query like the ones we already use in rhobs/configuration would be a good idea here? 🙂